.github/workflows/deployment.yml

```diff
@@ -74,9 +74,27 @@ jobs:
           oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_TOOL_TOKEN }}"
           GIT_BRANCH=${GITHUB_HEAD_REF} bash openshift/scripts/oc_build.sh ${SUFFIX} apply
 
+  build-web-image:
+    name: Build Web Image
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Set Variables
+        shell: bash
+        run: |
+          echo "SUFFIX=pr-${{ github.event.number }}" >> $GITHUB_ENV
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Build Image
+        shell: bash
+        run: |
+          oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_TOOL_TOKEN }}"
+          GIT_BRANCH=${GITHUB_HEAD_REF} NAME_APP=wps-web DOCKER_FILE=Dockerfile.web PATH_BC=openshift/templates/build.web.bc.yaml bash openshift/scripts/oc_build.sh ${SUFFIX} apply
+
   deploy-dev:
     name: Deploy to Dev
-    needs: [build-image, prepare-dev-database]
+    needs: [build-image, build-web-image, prepare-dev-database]
```

Comment:
I presume `build-image` refers to the backend image? And then `deploy-dev` means deploying both back- & front-end images, is that correct?

Reply:
Good point. I'll rename "build-image" to "build-api-image". This story doesn't look at deploying yet - so I'll leave "deploy-dev" alone for now.